packages/search-ui-workplace-search-connector/src/__tests__/WorkplaceSearchAPIConnector.test.ts

```diff
@@ -331,6 +334,55 @@ describe("WorkplaceSearchAPIConnector", () => {
       `);
     });
 
+    it("will include meta header information correct for the browser context", async () => {
+      const state = {
+        ...DEFAULT_STATE,
+        searchTerm: "searchTerm",
+        current: 1,
+        resultsPerPage: 10,
+        sortDirection: "desc" as const,
+        sortField: "name",
+        filters: [
+          {
+            field: "title",
+            type: "all" as const,
+            values: ["Acadia", "Grand Canyon"]
+          },
+          {
+            field: "world_heritage_site",
+            values: ["true"],
+            type: "all" as const
+          }
+        ]
+      };
+
+      const queryConfig = {
+        current: 2,
+        resultsPerPage: 5,
+        sortDirection: "asc" as const,
+        sortField: "title",
+        filters: [
+          {
+            field: "date_made",
+            values: ["yesterday"],
+            type: "all" as const
+          }
+        ]
+      };
+
+      await subject(state, queryConfig);
```

Comment:
Maybe we could replace this with just `await subject();`? It looks like `state` and `queryConfig` values are not important for this test case.